Source/Adobe.Target.Client/Util/TargetConstants.cs

```diff
@@ -53,9 +53,9 @@ public static class TargetConstants
         public const string AamBlob = "MCAAMB";
 
         /// <summary>
```

Comment:
`TargetConstants` is public, we should be careful when we rename fields in public classes, unless we want to make this `internal`.

Reply:
right, I'll revert it

Reply:
reverted